WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -229,7 +210,7 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
                     ImmuTableSection(rows: [
                         appSettingsRow,
                         ]),
-                    helpAndSupportSection,
+                    ImmuTableSection(rows: [helpAndSupportIndicator]),
```

Comment:
I started to ask if we should change the formatting to match the example above but I actually think the compact single line is better. :)